src/cli.ts

```diff
@@ -101,21 +101,28 @@ function loadProjectId(cmd: Command) {
 function validateOptions(cmd: Command) {
   const opts = cmd.optsWithGlobals();
 
-  if (!opts.apiKey) {
-    exitWithError(
-      'No API key has been provided. You must either provide one via --api-key, or login via `tolgee login`.'
-    );
-  }
-
   if (opts.projectId === -1) {
     error(
       'No Project ID have been specified. You must either provide one via --project-id, or by setting up a `.tolgeerc` file.'
     );
+    info(
+      'If you provide Project Api Key (PAK) via `--api-key`, Project ID is derived automatically.'
+    );
     info(
       'Learn more about configuring the CLI here: https://tolgee.io/tolgee-cli/project-configuration'
     );
     process.exit(1);
   }
+
+  if (!opts.apiKey) {
+    error(
+      `Not authenticated for host ${ansi.blue(opts.apiUrl.hostname)} and project ${ansi.blue(opts.projectId)}.`
+    );
+    info(
+      `You must either provide api key via --api-key or login via \`tolgee login\` (for correct api url and project)`
```

Comment:
```suggestion
      `You must either provide api key via --api-key or login via \`tolgee login\` for specific API URL and project`
```

Reply:
The CLI should also have a command which prints out for which projects (with their names) is the user currently logged in. I would also print this list when this error is printed out.

Reply:
Ok, done